Source/Infrastructure/ShoKanri.DAO/Repositories/UnitOfWork.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using ShoKanri.DAO.Context;
+using ShoKanri.Domain.Contracts.Data.Services;
+
+namespace ShoKanri.DAO.Repositories;
+
+public class UnitOfWork : IUnitOfWork
+{
+    private readonly AppDbContext _context;
+
+    public UnitOfWork(AppDbContext context)
+    {
+        _context = context;
+    }
+    public async Task CommitAsync()
+    {
+        await _context.SaveChangesAsync();
+    }
+}
```

Comment:
```suggestion
public class UnitOfWork(AppDbContext context) : IUnitOfWork
{
    public async Task CommitAsync()
        => await context.SaveChangesAsync();
}
```

Vale para essa e para outras classes: dê preferência ao **Primary Constructor**.